src/API/Site/Controllers/Ads/RecommendationsController.php

```diff
@@ -103,10 +116,20 @@ protected function get_recommendations_callback(): callable {
 				/** @var AdsRecommendationsService $query */
 				$query = $this->container->get( AdsRecommendationsService::class );
 
-				$type = $request->get_param( 'type' ) ?? 'IMPROVE_PERFORMANCE_MAX_AD_STRENGTH';
-				$id   = (int) $request->get_param( 'id' );
+				$type = $request->get_param( 'type' ) ?? [ 'IMPROVE_PERFORMANCE_MAX_AD_STRENGTH' ];
```

Comment:
@mukeshpanchal27 the `type` parameter should be required and not have a default of `IMPROVE_PERFORMANCE_MAX_AD_STRENGTH`.

If the type parameter is missing in the request we should either return an error or return an empty array.